### ISSUE TITLE
Never opens

### ISSUE THREAD

**Issue by Gorusss** [six7/figma-batch-styler#7]
Hi, Jan. 
I have a file with 69 text styles, 73 color styles, 5 effect styles and 5 grid styles. 

Well... as much as I'm waiting for your plugin, it won't open, **even if I restart it**. It's a real shame because I could really use it. 

Greetings. I wish you the best.

**Comment by six7** [comment]
Hey @Gorusss 

could you maybe duplicate that file for me and send me a link to it to see if i can find out what causes this? I've had this happen in the past, but now I load all the styles after launching the plugin (back then the problem was that fonts took quite a while to load, which we require for changing font families)

Also, I just pushed an update that waits with loading all styles for a bit, to possibly eliminate any issues there. However, I'm not sure that fixes the problem.

**Comment by Gorusss** [comment]
Hello. Still not working. 
I have no problem sharing the file with you, but I don't really know how to do it. I'd need your e-mail to get you to be the editor. Anyway, this is the file:
https://www.figma.com/file/7s5VFDt0OxzqsYdBFU5AgQ/19073-Living-Alternatives?node-id=1%3A2?.

**Comment by six7** [comment]
Hey @Gorusss - thanks for sharing that file, I think I figured out the problem.

I wasn't able to load styles as well with your file - for me the loading spinner showed up.

I fixed a small bug concerning gradient color styles - and with that fix I was able to load all 69 text styles and 72 color styles that batch styler supports.

Give it a try, already published to Figma. I think you'd need to restart Figma for the update to take effect.
